gbif-occurrence-linkage/requirements.txt

```diff
@@ -1,2 +1,4 @@
-kafka-python==2.0.2
-requests==2.31.0
+requests==2.32.3
+Pillow==11.2.1
```

Comment:
I don't think you need pillow for the gbif MAS

Reply:
:+1: